docs/en/about-appium/1.x-eol.md

```diff
@@ -0,0 +1,40 @@
+# Appium version 1 End Of Life (EOL) Plan
+
+Appium 1 has been around for quite a while and now is the time to move
+project development forward and to provide even better automation
+experience with Appium version 2.
+
+## Upgrading to Appium 2.x
+
+While Appium 2.0 leaves most of the fundamental automation behaviors from the various platfrom
+drivers untouched, it does rework the way that drivers are installed and accessed, and enforces
+stricter protocol usage, among other changes. For that reason, it's important to fully read,
+understand, and implement the steps discussed in the [Appium 1.x to 2.x Migration
+Guide](../advanced-concepts/migrating-to-appium-2.0.md). There you will find a full list of the
+changes, their rationale, and what you will need to do to keep your tests running well.
+
+## Transition Plan
+
+Appium 2.0 is currently in beta and is ready for your use and testing. Please begin the process of
+ensuring your test suites, or any other software built on Appium, is compatible with Appium 2.0.
+
+The tentative official release date of Appium 2.0 is *September 15, 2021*. After the release of
+Appium 2.0 (whether it is on that date or not), we will not be making any feature additions to the
+Appium 1.x server. We will however make the following kinds of updates for 6 months following the
```

Comment:
Btw, the release date of Appium 1.22 depends on iOS15/Xcode 13 for now. So, we can address that the 1.22 will be the last release as 1.x here as well. (The next 1.x will be only critical issue fixes then)